mak/copyimports.d

```diff
@@ -0,0 +1,40 @@
+/**
+ * Helper script to copy source files to the `import` directory
+ *
+ * When building druntime, files in `src` are copied to the `import` directory, excluding modules from the `rt` package.
```

Comment:
Nit: 3 things are excluded, `rt.*`, `gc.*` and `test_runner.d`. And the .d -> .di conversions in `core.sync.*` as you've mentioned (LDC doesn't do that).

Reply:
Slightly adapted the comment, but left out the di-conversion as I plan to copy these files, too.